dss-api.yml

```diff
@@ -71,20 +71,17 @@ paths:
           type: string
       responses:
         200:
-          description: OK
+          description: All results retrieved.
           schema:
-            type: object
-            properties:
-              es_query:
-                description: Elasticsearch query used to produce the results.
-                type: object
-              results:
-                description: Results matching the `es_query`.
-                type: array
+            $ref: '#/definitions/SearchResult'
+        206:
+          description: Page of result retrieved.
+          schema:
+            $ref: '#/definitions/SearchResult'
           headers:
             Link:
               type: string
-              description: URL to retrieve paginated results in [RFC 5988](https://tools.ietf.org/html/rfc5988) format.
+              description: URL to retrieve paginated results.
```

Comment:
URL to retrieve the next page of results (?)

Reply:
Will fix